mlir/lib/QEC/Transforms/clifford_t_to_ppr.cpp

```diff
@@ -46,6 +46,8 @@ struct CliffordTToPPRPass : impl::CliffordTToPPRPassBase<CliffordTToPPRPass> {
         target.addLegalOp<quantum::DeviceInitOp, quantum::DeviceReleaseOp>();
         target.addLegalOp<quantum::AllocOp, quantum::DeallocOp>();
         target.addLegalOp<quantum::InsertOp, quantum::ExtractOp>();
+        target.addLegalOp<quantum::NamedObsOp>();
+        target.addLegalOp<quantum::ExpvalOp>();
```

Comment:
I don't think we want to allow expectation values at this stage, we don't really have a strategy to handle complex measurement processes.

Reply:
Right, so I don't think we need to allow them in the input :)